python/cudf/cudf/tests/test_csv.py

```diff
@@ -1366,7 +1367,10 @@ def test_csv_reader_column_names(names):
         assert list(df) == list(names)
 
 
-@pytest.mark.xfail(reason="https://github.com/rapidsai/cudf/issues/10618")
+@pytest.mark.xfail(
+    condition=PANDAS_LT_140,
```

Comment:
This is a good question. CI currently tests with the latest pandas version that satisfies our dependency pinning. These version-conditional statements are all throughout our tests but I strongly doubt that any non-latest version would actually pass our tests, since we never verify that. I would recommend we discuss this in our next Python meeting. @quasiben ran into this before and we had a conversation about it here: https://github.com/rapidsai/cudf/pull/12150

Reply:
Ah this looks like almost exactly the same comment that I had in my review. I agree, let's discuss it in the next meeting and see if we can remove these.